appdaemon/events.py

```diff
@@ -300,6 +301,15 @@ def has_log_callback(self, name):
                                                                             "pin_thread": callback["pin_thread"],
                                                                             "kwargs": callback["kwargs"]
                                                                         })
+                                
+                                # Remove the callback if appropriate
+                                if executed is True:
+                                    remove = callback["kwargs"].get("oneshot", False)
+                                    if remove is True:
+                                        removes.append({"name": name, "uuid": uuid_})
+
+        for remove in removes:
```

Comment:
Take care that you are overwriting `remove`. It is probably better to use a different variable name.

Reply:
Well I could, but `remove` is really of no serious use, so I could use it.

Works anyway ;)

Reply:
What do you mean, no serious use?